.github/workflows/automate_changeset_feedback.yml

```diff
@@ -1,7 +1,7 @@
 name: Automate changeset feedback
 on:
   pull_request_target:
-    branches: ['main']
+    branches: ["main"]
```

Comment:
We have used single quotes exclusively - are we missing a prettierrc file? Or are we changing the standard here?